eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/BatchingEventStorageEngine.java

```diff
@@ -225,6 +225,8 @@ protected void validate() throws AxonConfigurationException {
 
```

Comment:
I am missing a test case for this change. Would you mind adding one @krosenvold?